### ISSUE TITLE
Extract Common Test Helpers and Properties

### ISSUE THREAD

**Issue by Dzol** [esl/jerboa#15]
Some helpers reside in the [helper](https://github.com/esl/jerboa/tree/master/test/helper) directory.

Where there are common helper functions, e.g. repeated across test files, let's move them under this directory. They could potentially help to isolate and build property test generators too.

**Comment by arkgil** [comment]
Yup, when [this](https://github.com/pragdave/pollution/pull/5) is merged into pollution, we could extract most of our generators and remove most of custom checks.

**Comment by arkgil** [comment]
The PR into Pollution got merged, we can work on extracting properties too now.
